Azure/azure-osconfig
# Moved Debian packaging content to DevOps folder

## Description

Added Debian packaging scripts and README.MD to DevOps folder

## Checklist

- [X] I have merged the latest `main` branch prior to this PR submission.
- [X] I submitted this PR against the `main` branch. 

## Validation Steps

- sudo cmake --build . --config release --target package (generate osconfig .deb file)
- sudo dpkg -i osconfig_<version>_bionic_x86_64.deb (install osconfig)
- sudo dpkg -l '\*osconfig\*' (confirm osconfig is installed as expected)
- sudo dpkg -P osconfig (uninstall osconfig and purge associated config files)
- sudo dpkg -l '\*osconfig\*' (confirm osconfig is uninstalled as expected)
